content/patterns/radio/examples/radio-rating.html

```diff
@@ -234,14 +232,14 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <th scope="row"><code>tabindex="0"</code></th>
               <td><code>g</code></td>
               <td>
-                <ul>
-                  <li>Makes the radio button focusable and includes it in the page <kbd>Tab</kbd> sequence.</li>
-                  <li>Set on only one radio in the radio group.</li>
-                  <li>On page load, is set on the first radio button in the radio group.</li>
-                  <li>Moves with focus inside the radio group so the most recently focused radio button is included in the page <kbd>Tab</kbd> sequence.</li>
+<ul>
+                  <li>ラジオボタンをフォーカス可能にし、ページの <kbd>Tab</kbd> シーケンスに含める。</li>
+                  <li>ラジオグループ内で、一つのラジオボタンにのみ設定される。</li>
+                  <li>ページの読み込み時に、ラジオグループ内の最初のラジオボタンに設定される。</li>
+                  <li>ラジオグループ内でフォーカスが移動すると、最近にフォーカスされたラジオボタンがページの <kbd>Tab</kbd> シーケンスに含まれるようになる。</li>
```

Comment:
ここは、withの訳出がニュアンス上重要なように思いましたがいかがでしょうか？

```suggestion
                  <li>ラジオグループ内でフォーカスとともに移動するため、最近にフォーカスされたラジオボタンがページの <kbd>Tab</kbd> シーケンスに含まれるようになる。</li>
```

Reply:
原文を改めて確認しました。おっしゃるように、with の訳出がうまくできていませんでした。異論ありません！

> Moves **with** focus inside the radio group so the most recently focused radio button is included in the page Tab sequence.